.buildkite/premerge.steps.yaml

```diff
@@ -0,0 +1,30 @@
+---
+common: &common
+  agents:
+    - "agent_count=1"
+    - "boot_disk_size_gb=400"
```

Comment:
This label should not be necessary either.

Reply:
removed